frontend/public/components/routes/create-route.tsx

```diff
@@ -517,20 +530,21 @@ export class CreateRoute extends React.Component<{}, CreateRouteState> {
                     name="secure"
                     aria-describedby="secure-help"
                   />
-                  Secure route
+                  {t('create-route~Secure route')}
```

Comment:
```suggestion
                  {t('create-route~Secure Route')}
```

Reply:
The old test script frontend/__tests__/components/routes/create-route.spec.tsx could not work after create-route.tsx chaged .
The error message:
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_console/7670/pull-ci-openshift-console-master-frontend/1348843560975208448,for example:
TypeError: Cannot read property 'contains' of undefined
    at Object.it (/go/src/github.com/openshift/console/frontend/__tests__/components/routes/create-route.spec.tsx:35:20)
    at Object.asyncFn (/go/src/github.com/openshift/console/frontend/node_modules/jest-jasmine2/build/jasmine_async.js:124:345)
    at resolve (/go/src/github.com/openshift/console/frontend/node_modules/jest-jasmine2/build/queue_runner.js:46:12)
    at new Promise (<anonymous>)
    at mapper (/go/src/github.com/openshift/console/frontend/node_modules/jest-jasmine2/build/queue_runner.js:34:499)
    at promise.then (/go/src/github.com/openshift/console/frontend/node_modules/jest-jasmine2/build/queue_runner.js:74:39)
    at process._tickCallback (internal/process/next_tick.js:68:7)

Reply:
Updated create-route.spec.tsx and create-route.tsx ,fix the issue. The create-route.spec.tsx works.

Reply:
Done